src/Menu/Menu.component.tsx

```diff
@@ -3,119 +3,147 @@ import * as React from 'react';
     // Color
     background?: string;
     color?: string;
     // Configuration
     justify?: 'flex-start' | 'flex-end';
 }
 
+const MenuSizeStyles = {
+    sm: {
+        menu: css``,
+        items: css`
+            font-size: 0.75rem;
+            line-height: 0.75rem;
+            font-weight: 500;
+            padding: 0.5rem 0.75rem;
+        `,
+    },
+    md: {
+        menu: css``,
+        items: css`
+            border-radius: base;
+            font-weight: 600;
+            padding: 1rem 0.75rem;
+        `,
+    },
+    lg: {
+        menu: css``,
+        items: css`
+            border-radius: base;
+            font-weight: 600;
+            padding: 1.5rem 1.25rem;
+        `,
+    },
+};
+
+const sizeVariant = variant({
+    key: 'menu.sizes',
+    prop: 'size',
+    default: 'md',
+    variants: MenuSizeStyles,
+});
+
 const StyledMenu = styled('nav')<MenuProps>`
-  margin: 0;
-  padding: 0;
-  display: flex;
-  width: 100%;
-  min-width: 250px;
-
-  ${({ background = 'primary.base' }) =>
-      css({
-          background: th.color(background),
-      })};
-  font-size: 0.875rem;
-  font-family: ${th('typography.fontFamily')};
-  border-radius: base;
-
-  /* TODO: use React.clone to append a run-time class here instead of using a * selector */
-  > * > a,
-  > a {
-    display: inline-block;
-    line-height: 1.125rem;
-    ${({ color = 'neutrals.white.base' }) =>
+    display: flex;
+    width: 100%;
+    ${spaceStyles};
+    min-width: 15.625rem;
+    margin: 0;
+    padding: 0;
+
+    ${({ background = 'primary.base' }) =>
         css({
-            color,
+            background: th.color(background),
         })};
-    transition: background-color 500ms, color: 500ms;
-    margin: 0 0.125rem;
-
-    &:hover {
-      ${({ color }) =>
-          css({
-              color: color ? lighten(20, color) : 'neutrals.white.base',
-          })};
-      background-color: rgba(255, 255, 255, .1);
-    }
+    font-size: 0.875rem;
+    font-family: ${th('typography.fontFamily')};
+    border-radius: base;
 
-    &:active,
-    &.active {
-      color: ${({ color }) =>
-          css({
-              color: color ? lighten(20, color) : 'neutrals.white.base',
-          })};
-      background-color: rgba(255, 255, 255, .1); // TODO: more or less opacity
-    }
+    .anchor-menu-item {
+        line-height: 1.125rem;
+        ${({ color = 'neutrals.white.base' }) => css({ color })};
+        transition: background-color 250ms, color 250ms;
+        margin: 0 0.125rem;
 
-    &:focus {
-      background-color: rgba(255, 255, 255, .1);
-    }
+        &:hover {
+            ${({ color }) =>
+                css({
+                    color: color ? lighten(20, color) : 'neutrals.white.base',
+                })};
+            background-color: rgba(255, 255, 255, 0.1);
+        }
 
-    &:first-of-type {
-      margin-left: 0;
-    }
+        &:active,
+        &.active {
+            color: ${({ color }) =>
```

Comment:
The initial `color: ` is redundant.

Reply:
👍

Reply:
I think you marked this resolved, but it hasn't been updated in `origin` yet